README.md

```diff
@@ -66,18 +66,39 @@ $ # Run the following from the root of the Hipcheck repository.
 $ docker build -t hipcheck:3.1.0 -f ./Containerfile
 ```
 
-### Build from Source
+### Local Install
 
 First, install the Rust compiler. We recommend following the official
 [installation instructions][install_rust]. Make sure to add
 `${CARGO_HOME}/bin` to your `PATH`.
 
-Next, get the Hipcheck repository and run `cargo xtask install`.
+You will also need Node installed. We recommend following the official
+[installation_instructions][install_node].
+
+You may either install using an [automated script](#script-based-installation)
```

Comment:
Should amend to "install Hipcheck" here since just was talking about Node last sentence

Reply:
Agree